src/_shared/js/capi-multiple.js

```diff
@@ -10,9 +10,9 @@ const ENDPOINT = 'https://api.nextgen.guardianapps.co.uk/commercial/api/capi-mul
 
 const OVERRIDES = {
     urls: ['[%Article1URL%]', '[%Article2URL%]', '[%Article3URL%]', '[%Article4URL%]'],
+    kickers: ['[%Article1Kicker%]', '[%Article2Kicker%]', '[%Article3Kicker%]', '[%Article4Kicker%]'],
```

Comment:
These vars don't exist in CAPI multiple paid for/supported, didn't you inject them using an additional param somewhere?